WalletWasabi/WabiSabi/Client/CoinJoin/Client/CoinJoinClient.cs

```diff
@@ -121,6 +121,11 @@ private async Task<RoundState> WaitForBlameRoundAsync(uint256 blameRoundId, Canc
 			throw new InvalidOperationException($"Blame Round ({roundState.Id}): Abandoning: the minimum output amount is too high.");
 		}
 
+		if (roundState.CoinjoinState.Parameters.MinInputCountByRound < CoinJoinConfiguration.AbsoluteMinInputCount)
+		{
+			throw new InvalidOperationException($"Blame Round ({roundState.Id}): Abandoning: the minimum input count was too low.");
+		}
```

Comment:
This is same than #13191 but now that I'm seeing it I wonder if we really need that... I need to test

Reply:
I've checked and now I'm sure we need it